__tests__/reducers/index.test.js

```diff
@@ -94,15 +96,15 @@ describe(`Takes a resource template ID and populates the global state`, () => {
       { "authenticate": {"loginJwt": {}},
         "lang": {"formData": []},
         "selectorReducer": { 'resourceTemplate:bf2:Monograph:Instance':
-          { 'http://id.loc.gov/ontologies/bibframe/instanceOf': { 'resourceTemplate:bf2:Monograph:Work': {} },
+          { 'http://id.loc.gov/ontologies/bibframe/instanceOf': {},
             'http://id.loc.gov/ontologies/bibframe/issuance':
               { items:
-                [ { id: {},
+                [ { id: 0,
```

Comment:
I assume it's ok for ids to crash across these arrays, but not within an array??

Reply:
Yes, keys only need to be unique within the context of the Javascript object; they won't clash across different arrays.